gradle.properties

```diff
@@ -1,4 +1,4 @@
-release_version = 2.0.5
+release_version = 3.0.0
```

Comment:
If we change the artifact's name, probably we should switch to version 1.0.0. Or we should not change the artifact/project name.
@Nikita-Smirnov-Exactpro I think we don't want to create a new artifact. What about you?

Reply:
Nikita said, that we shouldn't change the artifact name and increase major version (in this project), but in project which will be migrate (grpc-act and other), increase minor version